<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RaptorUploadingPage</class>
    <widget class="QWidget" name="RaptorUploadingPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>912</width>
                <height>548</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="_UploadingPageLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QSplitter" name="_Splitter">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="opaqueResize">
                        <bool>true</bool>
                    </property>
                    <property name="handleWidth">
                        <number>1</number>
                    </property>
                    <property name="childrenCollapsible">
                        <bool>false</bool>
                    </property>
                    <widget class="QTableView" name="_ItemView">
                        <property name="styleSheet">
                            <string notr="true"/>
                        </property>
                        <property name="verticalScrollBarPolicy">
                            <enum>Qt::ScrollBarAsNeeded</enum>
                        </property>
                    </widget>
                    <widget class="QWidget" name="_ItemAttributePanel" native="true">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <layout class="QGridLayout" name="_ItemAttributePanelLayout">
                            <property name="leftMargin">
                                <number>12</number>
                            </property>
                            <property name="topMargin">
                                <number>12</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>12</number>
                            </property>
                            <property name="spacing">
                                <number>12</number>
                            </property>
                            <item row="3" column="1">
                                <widget class="QLabel" name="_ItemSize">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QLabel" name="_ItemCreated">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="0">
                                <widget class="QLabel" name="_ItemSizeTip">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="6" column="1">
                                <widget class="QLabel" name="_ItemSpeed">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item row="5" column="0">
                                <widget class="QLabel" name="_ItemEstimatedTimeTip">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="5" column="1">
                                <widget class="QLabel" name="_ItemEstimatedTime">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QTextEdit" name="_ItemName">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="verticalScrollBarPolicy">
                                        <enum>Qt::ScrollBarAsNeeded</enum>
                                    </property>
                                    <property name="horizontalScrollBarPolicy">
                                        <enum>Qt::ScrollBarAlwaysOff</enum>
                                    </property>
                                    <property name="sizeAdjustPolicy">
                                        <enum>QAbstractScrollArea::AdjustToContents</enum>
                                    </property>
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="_ItemNameTip">
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="_ItemCreatedTip">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item row="9" column="1">
                                <spacer name="_ItemAttributePanelSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="6" column="0">
                                <widget class="QLabel" name="_ItemSpeedTip">
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>16777215</width>
                                            <height>26</height>
                                        </size>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true"/>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
